controllers/masteruserrecord/sync_test.go

```diff
@@ -645,6 +642,73 @@ func TestSynchronizeUserAccountFailed(t *testing.T) {
 	})
 }
 
+func TestRemoveAccountFromStatus(t *testing.T) {
+	// given
+	apiScheme(t)
+	userSignup := commonsignup.NewUserSignup()
+	userSignup.Status = toolchainv1alpha1.UserSignupStatus{
+		CompliantUsername: "john",
+	}
+
+	t.Run("remove UserAccount from the status when there is one item", func(t *testing.T) {
+		// given
+		mur := murtest.NewMasterUserRecord(t, "john",
+			murtest.WithOwnerLabel(userSignup.Name),
+			murtest.StatusCondition(toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, "")),
+			murtest.StatusUserAccount(test.MemberClusterName, toBeProvisioned()))
+		hostClient := test.NewFakeClient(t, mur, readyToolchainStatus, userSignup)
+		sync, _ := prepareSynchronizer(t, nil, mur, hostClient)
+
+		// when
+		err := sync.removeAccountFromStatus()
+
+		// then
+		require.NoError(t, err)
+		murtest.AssertThatMasterUserRecord(t, mur.Name, hostClient).
+			HasStatusUserAccounts().
+			HasConditions(toBeProvisioned(), toBeProvisionedNotificationCreated())
+	})
+
+	t.Run("remove UserAccount from the status when there are two items and align readiness", func(t *testing.T) {
+		// given
+		mur := murtest.NewMasterUserRecord(t, "john",
+			murtest.WithOwnerLabel(userSignup.Name),
+			murtest.StatusCondition(toBeNotReady("Provisioning", "")),
```

Comment:
Technically speaking - it doesn't matter what the values is, right? 
I think that it's OK to not always use constant in unit tests - it makes sure that it either works or doesn't work based on the actual value and not the value of the constant.

Reply:
Right, technically speaking it doesn't matter. 

it is just a minor detail, but using only the constants or the strings helps with consistency in case we need to search and replace this condition in the future with something else. For unit tests it's not a big deal since those should run very quickly allowing you to spot left overs immediately. 

Again just minor thing, I'm ok with keeping both strings and constants 👍